magento/magento2
# Simplified media gallery AssetDetailsProvider implementation

### Description (*)
Simplified media gallery AssetDetailsProvider implementation

### Fixed Issues (if relevant)
1. Fixes https://github.com/magento/adobe-stock-integration/issues/1726

### Manual testing scenarios (*)
1. Add images from Media Gallery to Page, Block, Category and Product content
2. Go to Media Gallery (check on both embedded (opened from wysiwyg) and standalone (Content -> Media Gallery menu) Media Gallery)
3. View details of the images - the "Used In" information in View Details panel should be displayed correctly
4. Delete the images - entities, where the image is used in, should be mentioned in the deletion warning popup

### Contribution checklist (*)
